src/chrome/content/rules/Zam.xml

```diff
@@ -1,30 +1,82 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://lkcss.zamimg.com/ => https://lkcss.zamimg.com/: Too many redirects while fetching 'https://lkcss.zamimg.com/'
-Fetch error: http://lkimg.zamimg.com/ => https://lkimg.zamimg.com/: Too many redirects while fetching 'https://lkimg.zamimg.com/'
-Fetch error: http://lkjs.zamimg.com/ => https://lkjs.zamimg.com/: Too many redirects while fetching 'https://lkjs.zamimg.com/'
-Fetch error: http://static.wowhead.com/ => https://static.wowhead.com/: (51, "SSL: no alternative certificate subject name matches target host name 'static.wowhead.com'")
-Fetch error: http://www.destinydb.com/ => https://www.destinydb.com/: (60, 'SSL certificate problem: unable to get local issuer certificate')
+	Cert expired:
+		- www.destinydb.com
+
+	Cert mismatch:
+		- static.wowhead.com
+		- google.analytics.zam.com
+		- mb.zam.com, only valid for sv.monkeybroker.net
+		- ox-i.zam.com
+		- lolstatic.zam.com
+		- hearthstone.services.zam.com
+		- lolleaderboard.services.zam.com
+		- lolleague.services.zam.com
+		- lolriotstats.services.zam.com
+		- lolstats.services.zam.com
+		- lolsummoner-stats.services.zam.com
+		- lolsummoner.services.zam.com
+		- replay.services.zam.com
 
+		
+	Chain issues:
+		- aion.zam.com
+		- www.daoc.zam.com
+		- eq.zam.com
+		- www.eq.zam.com
+		- eve.zam.com
+		- eww.zam.com
+		- www.lotro.zam.com
+		- www.lualua.zam.com
+		- m.zam.com
+		- rift.zam.com
+		- www.rift.zam.com
+		- premium.services.zam.com
+		- sha.zam.com
+		- smtp.zam.com
+		- tera.zam.com
+		- wow.zam.com
+		- zameq2.zam.com
+
+	TLS error:
+		- logging.services.zam.com
 -->
-<ruleset name="Zam Network (partial)" default_off='failed ruleset test'>
+<ruleset name="Zam Network (partial)" >
+
+	<target host="aov.zam.com" />
+	<target host="eq2.zam.com" />
+	<target host="ffxiv.zam.com" />
+	<target host="fr.zam.com" />
+	<target host="*.g00.zam.com" />
+	<target host="secure.zam.com" />
+	<target host="auth.services.zam.com" />
+	<target host="cms.services.zam.com" />
+	<target host="comment.services.zam.com" />
+	<target host="favorite.services.zam.com" />
+	<target host="loldata.services.zam.com" />
+	<target host="media.services.zam.com" />
+	<target host="media-staging.services.zam.com" />
+	<target host="parkplace-staging.services.zam.com" />
 
 	<target host="cdn.zamimg.com"/>
 	<target host="descss.zamimg.com"/>
 	<target host="desimg.zamimg.com"/>
 	<target host="desjs.zamimg.com"/>
-	<target host="lkcss.zamimg.com"/>
```

Comment:
Can we keep a comment for it?

Reply:
both of these seem to no longer be in use on lolking and replaced by content on the .services subdomain

Reply:
But they are still accessible.

Reply:
mhmm i think i removed them for not having proper test urls and non 200 status codes or some sort of error like that, let me check that again

Reply:
You can just keep them as comments.